### PR TITLE
Add guide for fixture

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1715

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 3 min 3 sec

File: docs/src/developer/guide/templates/pull-request-template.md [hunk 1]
```diff
@@ -11,4 +11,39 @@ context_id: LmD2JcMLJGCcn6yEc4bvKW
...
 # Pull Request Template
-<!-- To Be Populated -->
\ No newline at end of file
+# Pull Request
+- Push the code on the GitHub.
+- Click on the `Compare & pull request` button to create a PR.
+## Description
+
+- Describe the purpose and context of the changes made in this PR.
+- Add screenshots which claryfies the implemented feature, fixed bug etc.
```
Comment (Jeremy-Prior) [inline docs/src/developer/guide/templates/pull-request-template.md:20]:
```suggestion
- Add screenshots which clarifies the implemented feature, fixed bug etc.
```

File: docs/src/developer/guide/templates/pull-request-template.md [hunk 2]
```diff
@@ -11,4 +11,39 @@ context_id: LmD2JcMLJGCcn6yEc4bvKW
 ---
 
 # Pull Request Template
-<!-- To Be Populated -->
\ No newline at end of file
+# Pull Request
```
Comment (Jeremy-Prior) [inline docs/src/developer/guide/templates/pull-request-template.md:14]:
```suggestion
# Pull Request Template

## Make Pull Request
```

File: docs/src/developer/guide/templates/pull-request-template.md [hunk 3]
```diff
@@ -11,4 +11,39 @@ context_id: LmD2JcMLJGCcn6yEc4bvKW
...
+- Push the code on the GitHub.
```
Comment (Jeremy-Prior) [inline docs/src/developer/guide/templates/pull-request-template.md:15]:
```suggestion
- Push the code on GitHub.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]
Minor changes

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 9 sec

**Comment by Jeremy-Prior** [comment]
Not comfortable merging as I'm not sure what is causing the django test to fail.

@dimasciput please review when possible.

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 17 sec

**Pull request merged**